finagle-core/src/test/scala/com/twitter/finagle/loadbalancer/TrafficDistributorTest.scala

```diff
@@ -227,7 +228,7 @@ class TrafficDistributorTest extends FunSuite {
       }.head
 
       result.foreach {
-        case ((w, _, l)) => assert(math.abs(l / w - baseline) <= baseline * ε)
+        case ((w, _, l)) => assert(math.abs(l / w - baseline) <= baseline * ε, "second one")
```

Comment:
This test is failing, todo: figure out why (and fix it)

Reply:
Increasing the size for R makes the test pass. I'm not sure if there is an underlying problem here, or that it was just unlucky RNG from the given seed.